frontend/src/app/containers/ExperimentPage.js

```diff
@@ -276,10 +276,11 @@ export class ExperimentDetail extends React.Component {
                 <h2 className="banner">
                   <span data-l10n-id="experimentPromoHeader" className="block">Ready for Takeoff?</span>
                 </h2>
-                <p data-l10n-id="experimentPromoSubheader">We're building next-generation features for Firefox. Install Test Pilot to try them!</p>
+                <p data-l10n-id="experimentPromoSubheader">We&apos;re building next-generation features for Firefox. Install Test Pilot to try them!</p>
```

Comment:
This looks strange: there's a change in the template but not in the FTL.

Is the text still localized if you switch to a different locale, like Italian or German? If it's not, I would suggest to drop this change.